Taarafo.Core/Brokers/Storages/StorageBroker.Posts.cs

```diff
@@ -22,7 +22,7 @@ public async ValueTask<Post> InsertPostAsync(Post post)
                 new StorageBroker(this.configuration);
 
             EntityEntry<Post> postEntityEntry =
-                await broker.Posts.AddAsync(post);
+                await broker.Posts.AddAsync(entity:post);
```

Comment:
Space is required. 'entity: post'. Maybe the running code cleanup will take care of this. Do verify in other places too.